app/proguard-rules/picasso-rules.pro

```diff
@@ -1,7 +1,2 @@
--keep class com.squareup.picasso.** { *; }
--keepclasseswithmembers class * {
-    @com.squareup.picasso.** *;
-}
--keepclassmembers class * {
-    @com.squareup.picasso.** *;
-}
+#Picasso
+-dontwarn com.squareup.okhttp.**
```

Comment:
I suspect the need came from the rule changes required because of OkHttp3. I added the rules one by one after seeing what the next issue was.